### PR TITLE
Edit file for readability, style, grammar.

### PR DESCRIPTION
**Pull request by WWitman**
This PR includes:

Addition of contextual content where needed for clarity
Structural changes as needed for clarity and consistency
Grammar, spelling, and other edits

Signed-off-by: Will Witman willwitman@google.com

### CODE REVIEW

File: content/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 1]
```diff
@@ -22,29 +20,51 @@ two steps: 50%, 100%.
 
 * Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
 
-## Weight-based version routing
+* Review the [Traffic Management](/docs/concepts/traffic-management) concepts doc.
 
-1.  Set the default version for all microservices to v1.
+* (Optional) Do the [Configuring request routing](/docs/tasks/traffic-management/request-routing) task.
```
[the remark this answers is not in the document]
Comment (WWitman) [reply content/docs/tasks/traffic-management/traffic-shifting/index.md]: I'll remove this.

File: content/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 2]
```diff
@@ -22,29 +20,51 @@ two steps: 50%, 100%.
...
     $ istioctl replace -f @samples/bookinfo/networking/virtual-service-reviews-50-v3.yaml@
     {{< /text >}}
 
-    Confirm the rule was replaced:
+    Wait a few seconds for the new rules to propagate.
+
+1. Confirm the rule was replaced:
 
-    {{< text bash yaml >}}
+    {{< text bash >}}
```
[the remark this answers is not in the document]
Comment (WWitman) [reply content/docs/tasks/traffic-management/traffic-shifting/index.md]: I see -- I didn't notice that you can still copy the command with the copy button even when combined this way. I'll change this too.

File: content/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 3]
```diff
@@ -22,29 +20,46 @@ two steps: 50%, 100%.
...
     {{< /text >}}
 
-    Confirm the rule was replaced:
+    Wait a few seconds for the new rules to propagate.
+
+1. Confirm the rule was replaced:
 
     {{< text bash yaml >}}
     $ istioctl get virtualservice reviews -o yaml
+
```
Comment (sdake) [inline content/docs/tasks/traffic-management/traffic-shifting/index.md]: linter complaining about this line.

**Pull request merged**